src/renderer/renderer.ts

```diff
@@ -943,7 +914,7 @@ export class SDFGRenderer {
 
         // If we're in a VSCode context, we also want to refresh the outline.
         if (this.in_vscode)
-            outline(this, this.graph);
+            SDFV.get_instance().outline(this, this.graph);
```

Comment:
can't we propagate the instance in the ctor? Alternatively, why is the `outline` method not static? The instance shouldn't matter